src/mame/namco/namcos10.cpp

```diff
@@ -3585,8 +3640,8 @@ GAME( 2000, mrdrilr2u, mrdrilr2, ns10_mrdrilr2,  mrdrilr2,     namcos10_memm_sta
 
 // MEM(N)
 GAME( 2000, gahaha,    0,        ns10_gahaha,    gahaha,       namcos10_memn_state,  memn_driver_init, ROT0, "Namco / Metro", "GAHAHA Ippatsudou (World, GID2 Ver.A)", MACHINE_IMPERFECT_SOUND )
-GAME( 2000, ptblank3,  0,        ns10_ptblank3,  namcos10,     namcos10_memn_state,  memn_driver_init, ROT0, "Namco", "Point Blank 3 (World, GNN2 Ver.A)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION ) // needs to hookup gun IO
-GAME( 2000, gunbalina, ptblank3, ns10_ptblank3,  namcos10,     namcos10_memn_state,  memn_driver_init, ROT0, "Namco", "Gunbalina (Japan, GNN1 Ver.A)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION )
+GAME( 2000, ptblank3,  0,        ns10_ptblank3,  ptblank3,     namcos10_memn_state,  memn_driver_init, ROT0, "Namco", "Point Blank 3 (World, GNN2 Ver.A)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION )
+GAME( 2000, gunbalina, ptblank3, ns10_ptblank3,  ptblank3,     namcos10_memn_state,  memn_driver_init, ROT0, "Namco", "Gunbalina (Japan, GNN1 Ver.A)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION )
```

Comment:
Can these be changed to just `MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION` flags?

Reply:
The game itself seems to play fine but there's a noticeable issue with the intro movie. It looks like a slideshow almost. I don't know if that'd hold it back from promotion or not.

Reply:
Do you have a relevant t10mmc log about disc data reads?

Reply:
It's not a CD game.

Reply:
Nevermind, I saw a PS1 video on Youtube. The arcade version just looks like a slideshow.
https://www.youtube.com/watch?v=5mZ5WrbJ75o

Reply:
I guess thru a DMA then?

>  I don't know if that'd hold it back from promotion or not.

Intro FMV is not relevant for promotion, assuming it don't crash emulation/emulator if left idle.